spec/system/candidate_interface/candidate_reviewing_application_for_course_unavailable_on_apply_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
```

Comment:
It might make more organisational sense to consolidate this spec with `spec/system/candidate_interface/candidate_reviewing_application_with_unavailable_course_options_spec.rb`, perhaps by adding a couple of steps which swap out one invalid course option for another and asserting the presence of the new warning.